src/OrchardCore/OrchardCore.ContentManagement.Abstractions/IContentPickerResultProvider.cs

```diff
@@ -21,6 +21,7 @@ public class ContentPickerSearchContext
     public class ContentPickerResult
```

Comment:
Can you move and extract these classes into separate files? I'm pretty sure they don't belong in the abstraction project.